scene/gui/range.h

```diff
@@ -53,6 +53,8 @@ class Range : public Control {
 
 	Shared *shared = nullptr;
 
+	bool allow_non_finite = false;
```

Comment:
I'm guessing this should be added to the `Shared` structure to be consistent with `allow_greater` and `allow_lesser`.